### DISCUSSION TITLE
Using controller rotation to move? (for proper FPS movement)

### DISCUSSION THREAD

**Discussion by adamfulop** [Sixze/ALS-Refactored#585] [Q&A]
Hey there, I'm trying to use ALS refactored for an FPS game (and share anim logic between the player and the NPCs) but having an issue that character turning is smoothed which makes FPS character that skating/sliding feel that is very unpleasant, especially during sprint. I've been wondering if there's any option to turn that off and always force the velocity in the view direction?

**Comment by ameaninglessname** [comment]
Looks like you are looking for `RotationInterpolationSpeedCurve` in movement settings : `MS_Als_Normal`

**Reply by adamfulop** [reply]
Together with the accel/decel/friction curve it did exactly what I needed, thank you!
